cpp/gazelle-cpp/compute/substrait_arrow.cc

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Could we make a new directory to put the files only related to arrow backend? In this way, each backend will have its own directory, and share the common code in parent directory.

Reply:
cpp/gazelle-cpp is for arrow's backend. You can create folders at the same level for other backend. For example cpp/velox.